scripts/deployEscrowAndStaking/deploy_transparent.js

```diff
@@ -16,6 +16,7 @@ const fs = require('fs');
 const { getTargetAddress, setTargetAddress, encodeCall } = require('../helpers');
 
 const user_key1 = process.env.PRIVATE_KEY;
+const ISSUER_ADDR = '0x42d9ac3ebebb9479f24360847350b4F7EADECE50';
```

Comment:
cant we get this from SNX library like we get proxySUSD and SafeMath?

Reply:
its a different address per chain

Reply:
done